provider/internal/connector/sdc/data_source_test.go

```diff
@@ -25,7 +31,8 @@ func TestAccSdcDeviceDataSource(t *testing.T) {
 			{
 				Config: acctest.ProviderConfig() + testSdcConfig,
 				Check: resource.ComposeAggregateTestCheckFunc(
-					resource.TestCheckResourceAttr("data.cdo_sdc_device.test", "id", testSdc.TenantId),
+					resource.TestCheckResourceAttr("data.cdo_sdc_connector.test", "name", testSdc.Name),
```

Comment:
I don't see any production code changes that result in the filter being applied? 
Do these tests pass because the acceptance tenant only has a single SDC associated?

Reply:
There is no production code change because it was not properly tested (my bad), it was only checking for the `id` of the whole resource but it should also check individual `sdc` as well.